src/app/components/pool/action/add/pool-create-basic-dialog.html

```diff
@@ -29,18 +29,19 @@
         </bl-form-section>
 
         <bl-form-section title="Select an operating system configuration" subtitle="Choose the operating system to be installed on all the nodes in the pool">
-            <bl-banner class="cloud-services-deprecation-warning" type="warning">
-                <div message>
-                    The Windows container images below will be removed from marketplace starting on September 22, 2022. After this date, new pools using it may not be created from the Azure Batch Explorer.
-                    <a (click)="openLink('https://github.com/Azure/Batch/issues/136')" href="javascript:void(0)">Click here </a>
-                    for more information.
-                </div>
-            </bl-banner>
+            <ng-container *ngIf="isSelectedContainerImageDeprecated">
+                <bl-banner class="create-pool-deprecation-warning" type="warning">
+                    <div message>
+                        Microsoft Windows Server container images are deprecated. Pools using this image will no longer work properly in Azure Batch Explorer after September 22, 2022.
+                        <a class="create-pool-warning-alternative-link" (click)="openLink('https://github.com/Azure/Batch/issues/136')" href="javascript:void(0)">Learn more </a>
+                    </div>
+                </bl-banner>
+            </ng-container>
             <ng-container *ngIf="cloudServiceConfiguration">
-                <bl-banner class="cloud-services-deprecation-warning" type="warning">
+                <bl-banner class="create-pool-deprecation-warning" type="warning">
                     <div message>
                         Cloud Services pools are deprecated and will be retired on 29 February 2024. Existing ‘CloudServiceConfiguration’ pools will need to be deleted and new
-                        <a class="cloud-services-warning-alternative-link" (click)="openLink('https://docs.microsoft.com/en-us/azure/batch/batch-pool-cloud-service-to-virtual-machine-configuration?WT.mc_id=Portal-Microsoft_Azure_Batch')" href="javascript:void(0)">‘VirtualMachineConfiguration’ pools created. </a>
+                        <a class="create-pool-warning-alternative-link" (click)="openLink('https://docs.microsoft.com/en-us/azure/batch/batch-pool-cloud-service-to-virtual-machine-configuration?WT.mc_id=Portal-Microsoft_Azure_Batch')" href="javascript:void(0)">‘VirtualMachineConfiguration’ pools created. </a>
```

Comment:
I realize this isn't actually new to this PR, but when linking to documentation, omit the locale (en-us in this case). Should be: https://docs.microsoft.com/azure/batch/batch-pool-cloud-service-to-virtual-machine-configuration?WT.mc_id=Portal-Microsoft_Azure_Batch